File: SiteVisitsNetCore2020Web/Models/Organization.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SiteVisitsNetCore2020Web.Models
{
    public class Organization
    {
        //public int Id { get; set; }
        public Guid Id { get; set; }
        public string Name { get; set; }
        public bool? IsIsp { get; set; }
        public DateTime FirstOccurrence { get; set; }
        public DateTime DateCreated { get; set; }
    }
}

File: SiteVisitsNetCore2020Web/ClientApp/src/app/models/Organization.ts
export class Organization {
  //id: number;
  public id: string;
  name: string;

  constructor(options: {
    //id: number,
    id: string,
    name: string
  }) {
    if (options) {
      this.id = options.id;
      this.name = options.name;
    }
  }
}
